charts/delphai-keda/values.yaml

```diff
@@ -29,3 +29,4 @@ env: []
 gpu: false
 mountPath: /app/data
 fileShares: []
+podAntiAffinity: false
```

Comment:
I would never guess what that config does. My only version is wrong — that I should put full k8s `podAntiAffinity` manifest here.

Lets find another name that would:
 - highlight that it's a boolean flag
 - describe that it really does
 - not be confused with usual k8s options

Reply:
True, I kept thinking about a better name but couldn't find a concise one that explains what it does. How about `repelsSimilarPods`?

Reply:
What is "similar" in this case? How similar should they be?

Maybe `onlyPodPerNode`?

Reply:
Maybe we should also add `PerApp` since we're using the `app` label and to not imply that it's one pod per node in general, so maybe `onlyPodPerNodePerApp` or `onePodPerNodePerApp` even though it's a bit long

Reply:
Sounds good

Reply:
`onePodPerNode` sounds like it will run a pod for each node (daemonset like), but it will run "not more that one" pod. I tried to express that with "only" instead of "one", but not sure if I succeeded

Reply:
Yeah, then `onePodPerNodePerApp` will do I believe

Reply:
One or only?

Reply:
Either is okey but `one` seems to be clearer than `only` when used with `PerNodePerApp`

Reply:
Ok sure, lets then finalize the PR and merge it